ado-build.yml

```diff
@@ -32,3 +27,23 @@ jobs:
       script: 'BuildAndTest.cmd'
 
   - task: ComponentGovernanceComponentDetection@0
+    inputs:
+      ignoreDirectories: '..\src\sarif-sdk'
```

Comment:
we already have component governance in sarif-sdk. We don't need to analyze here as well.

So, let's ignore the entire submodule.